src/schemas/FEEDBACK-TOOL/schema.js

```diff
@@ -92,7 +92,50 @@ const internationalSchoolAddress = {
   }
 };
 
-const schoolAddress = [domesticSchoolAddress, internationalSchoolAddress];
+// TRANSLATE: all search tool address fields into street, street2, and country
+const searchToolSchoolAddress = {
+  required: ['viaSearchTool', 'street', 'city', 'country'],
+  properties: {
+    viaSearchTool: {
+      type: 'boolean',
+      'enum': [
+        true
+      ]
+    },
+    country: {
+      type: 'string', // TYPE: text (255)
+      minLength: 1,
+      maxLength: 255
+    },
+    street: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 80
+    },
+    street2: {
```

Comment:
It seems like we should add a third street field to the manual and search international school address schemas, since the school search results feature addresses with three address lines, so as to provide a consistent experience for the user (manual entry supporting the same number of fields as the search tool). We can adjust the line length limits (80 --> 53), and include this as another translation that the backend can perform. Looping in @jholton for the backend perspective on this translation idea.

Reply:
So add a third street field to manual entry only for international addresses?

Reply:
Good question, we could/should also do this for domestic, that would be more consistent with the user experience we typically provide.

Reply:
Okay. That's easier. I just don't know when I've been presented with three different fields for street address before :) I guess that's more common outside of the US.

Reply:
@cehsu We can transform as necessary to fit the Salesforce requirements.